### PR TITLE
Add filtering interviews by date functionality and update help window

### PR DESCRIPTION
**Pull request by zhikaiong2001**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FilterInterviewsByDateCommand.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.model.Model;
+import seedu.address.model.interview.Interview;
+
+/**
+ * Filters the displayed interview list based on the date provided.
+ */
+public class FilterInterviewsByDateCommand extends Command {
```
Comment (headcube1) [inline src/main/java/seedu/address/logic/commands/FilterInterviewsByDateCommand.java:15]: Nice new command created! Helps with finding the correct interview!

**Review by headcube1 — COMMENTED** [review]
LGTM

**Review by headcube1 — APPROVED** [review]

File: src/main/java/seedu/address/logic/parser/FilterInterviewsByDateCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    /**
+     * Parses the given {@code String} of arguments in the context of FilterInterviewByDateCommand
+     * and returns a FilterInterviewByDateCommand object for execution.
+     * @throws ParseException if the user does not conform the expected date format
+     */
+    public FilterInterviewsByDateCommand parse(String args) throws ParseException {
+        try {
+            LocalDate date = LocalDate.parse(args.trim());
+            return new FilterInterviewsByDateCommand(date);
+        } catch (DateTimeParseException e) {
```
Comment (gingerbreaf) [inline src/main/java/seedu/address/logic/parser/FilterInterviewsByDateCommandParser.java:25]: Nice Try catch block! Good handling of exceptions.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 3]
```diff
@@ -26,7 +26,8 @@ public class HelpWindow extends UiPart<Stage> {
             + "\n5. List applicants/interviewers: list_persons"
             + "\n6. List interviews: list_interviews"
             + "\n7. Edit applicants/interviewers: edit [applicant/interviewer index] n/[newName]..."
-            + "\n8. Find person(s) by email/name/phone: find_[email/name/phone] [keyword1] [keyword2]...";
+            + "\n8. Find person(s) by email/name/phone: find_[email/name/phone] [keyword1] [keyword2]..."
+            + "\n9. Filter interview(s) by date: filter_interviews_by_date [date in YYYY-MM-DD]";
     public static final String HELP_MESSAGE = "Refer to our user guide at " + USERGUIDE_URL + " for detailed info "
```
Comment (gingerbreaf) [inline src/main/java/seedu/address/ui/HelpWindow.java:31]: Looks good

**Review by gingerbreaf — APPROVED** [review]
LGTM

**Pull request closed without merging**